index.js

```diff
@@ -38,7 +38,8 @@ module.exports = class s3proxy extends EventEmitter {
   createReadStream(req) {
     this.isInitialized();
     const r = s3proxy.parseRequest(req);
-    const params = { Bucket: this.bucket, Key: r.key };
+    let params = { Bucket: this.bucket, Key: r.key };
```

Comment:
Codacy has a fix for the issue: ['params' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/gmoon/s3proxy/pullRequest?prid=8021331)
```suggestion
    const params = { Bucket: this.bucket, Key: r.key };
```